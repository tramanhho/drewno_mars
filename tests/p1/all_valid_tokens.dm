and 24Kmagic bool else false too hot if int give take or perfect return class true today I don't feel like doing any work void while = : , + - == > >= { < <= ( ! != -- ++ } ) ; / *