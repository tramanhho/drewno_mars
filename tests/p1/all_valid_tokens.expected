AND [1,1]
MAGIC [1,5]
BOOL [1,14]
ELSE [1,19]
FALSE [1,24]
FALSE [1,30]
IF [1,38]
INT [1,41]
GIVE [1,45]
TAKE [1,50]
OR [1,55]
PERFECT [1,58]
RETURN [1,66]
CLASS [1,73]
TRUE [1,79]
EXIT [1,84]
VOID [1,123]
WHILE [1,128]
ASSIGN [1,134]
OTHER [1,136]
COMMA [1,138]
CROSS [1,140]
DASH [1,142]
EQUALS [1,144]
GREATER [1,147]
GREATEREQ [1,149]
LCURLY [1,152]
LESS [1,154]
LESSEQ [1,156]
LPAREN [1,159]
NOT [1,161]
NOTEQUALS [1,163]
POSTDEC [1,166]
POSTINC [1,169]
RCURLY [1,172]
RPAREN [1,174]
SEMICOL [1,176]
SLASH [1,178]
STAR [1,180]
EOF [1,182]